docs/api-usage.rst

```diff
@@ -205,11 +207,16 @@ listing methods support the ``page`` and ``per_page`` parameters:
    The first page is page 1, not page 0.
 
 By default GitLab does not return the complete list of items. Use the ``all``
-parameter to get all the items when using listing methods:
+parameter to get all the items when using listing methods. Alternatively, if
+the endpoint also accepts an ``all`` parameter itself, you can use the
+``all_pages`` parameter for listing methods to avoid this conflict:
 
 .. code-block:: python
 
    all_groups = gl.groups.list(all=True)
+   # or
+   all_groups = gl.groups.list(all_pages=True)
```

Comment:
Curious. Can they also use:
```
gl.groups.list(all=True, query_parameters={"all": True})
```

Curious, but also wondering if we should document that, if it does work.